docs/Manual/Security/Starter/README.md

```diff
@@ -0,0 +1,30 @@
+Securing Starter Deployments
+============================
+
+The password that is set for the _root_ user during the installation of the ArangoDB
+package has no effect in case of deployments done with the tool _ArangoDB Stater_,
+as this tool creates new database directories and configuration files that are
+separate from those created by the stand-alone installation.
+
+Assuming you have enabled authentication in your _Starter_ deployment, by default
+the _root_ user will be created with an _empty_ password.
+
+In order to the change the password of the _root_ user, you can:
+
+- Open the ArangoDB web UI and change the password from there. [More information](../Programs/WebInterface/Users.md).
+- Open an ArangoSH shell and use the function _users.replace_. [More information](../Administration/ManagingUsers/InArangosh.md#replace).
+
+In case you would like to automate the _root_ password change, you might use the 
+_--javascript.execute-string_ option of the _arangosh_ binary, e.g.:
+
+```
+arangosh --server.endpoint your-server-endpoint \
+	--server.password "" 
+	--javascript.execute-string 'require("org/arangodb/users").update("root", "mypwd");
+```
+
+where "mypwd" is the new password you want to set.
+
+If your _Starter_ deployment has authentication turned off, it is suggested to
```

Comment:
We should add a note saying that you cannot change authentication from on/off after you created the deployment once.

Reply:
This should work. However I prefer to keep this as a "don't do it like this" and implement changing the config files properly.